### ISSUE TITLE
Feature Request: Concentric Pie Charts

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#786]
```
What would you like to see us add to this API?
Support for Concentric Pie Charts would be excellent. Any plans to add this feature?


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?
Pie Charts

*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `sbaxter@precisionir.com` on 2012-01-11 17:14:21

**Comment by orwant** [comment]
```
This was possible using the old image charts API, that has since been deprecated. I
was counting on using it in a GAS "macro" I'm finally developing.
```

Original issue reported on code.google.com by `eric@schultzter.ca` on 2012-06-13 02:12:12

**Comment by swrobel** [comment]
:+1:

**Comment by rachelg** [comment]
this feature would be useful on the donut chart/pie chart particularly

**Comment by nbering** [comment]
@rachelg Have you seen [diff charts](https://developers.google.com/chart/interactive/docs/gallery/diffchart#diff-pie-charts)? They are kind of a concentric pie chart.

![image](https://cloud.githubusercontent.com/assets/4421371/16779176/244be84c-483f-11e6-83fb-02a09c90a64a.png)

This [pie chart in geo chart markers](http://jsfiddle.net/mba8grqb/1/) example also comes to mind. It's a bit of a hack where they draw the pie chart into the marker circle using `document.querySelectorAll()`.  I found that on [the google visualization forum](https://groups.google.com/forum/#!searchin/google-visualization-api/geochart$20pie/google-visualization-api/b1y174BOsi4/g1qtBFHtc44J). Maybe one could use a similar trick to fill the center circle of a donut chart, or overlay your own element on top of the chart and then with it with another pie.

![image](https://cloud.githubusercontent.com/assets/4421371/16779116/f4b4e336-483e-11e6-86bb-6d36f909ed8d.png)

**Comment by rachelg** [comment]
Yes, but I want multiple nested pies, 3 or 4, preferably separable ( like
that the circles are slightly apart, not directly beside one another). The
diff charts are unsuitable for my needs.

Thanks and regards

Rachel
